src/javascript/app/base/header.js

```diff
@@ -656,9 +656,10 @@ const Header = (() => {
         const isOAuth2Enabled = AuthClient.isOAuth2Enabled();
 
         if (isOAuth2Enabled) {
-            const redirectCallbackUri = `${window.location.origin}/en/callback`;
+            const currentLanguage = Language.get();
+            const redirectCallbackUri = `${window.location.origin}/${currentLanguage}/callback`;
             const postLoginRedirectUri = window.location.origin;
-            const postLogoutRedirectUri = `${window.location.origin}/en/trading`;
+            const postLogoutRedirectUri = `${window.location.origin}/${currentLanguage}/trading`;
             // Test commit
```

Comment:
Tested for `ru` and `cn_tw` language:

https://github.com/user-attachments/assets/547f4380-6c89-4004-81e1-8301322f6ae6

https://github.com/user-attachments/assets/779f0ea3-58c5-457f-8fa9-c2d64c9bc1d2